libethcore/KeyManager.cpp

```diff
@@ -79,7 +79,11 @@ bool KeyManager::load(string const& _pass)
 		bytes encKeys = contents(m_keysFile);
 		if (encKeys.empty())
 			return false;
+#ifdef TOLAR_TEST
+    m_keysFileKey = SecureFixedHash<16>(pbkdf2(_pass, salt, 1, 16));
+#else
```

Comment:
what does this 262144 mean? Why is changing it to 1 making it faster?

Reply:
It's a number of iterations/passes it makes when making hash.